eNMS/static/js/inventory.js

```diff
@@ -144,10 +144,18 @@ function handOffSSHConnection(id) {
     form: `connection-parameters-form-${id}`,
     callback: function(result) {
       let loc = window.location;
-      const link = `${result.username}@${loc.hostname}:${result.port}`;
-      const message = `Click here to connect to ${result.device_name}.`;
-      notify(`<a href='ssh://${link}'>${message}</a>`, "success", 15);
-    },
+      let url = settings.app.address;
+      if (!url) {
+        url = `${window.location.protocol}//${window.location.host}`;
+      }
```

Comment:
`url` is not actually used anywhere in the code that follows so I think it's safe to remove it

Reply:
I will.  I honestly don't recall why that's in there.  May have just overlooked it.